apps/sales/serializers.py

```
@@ -0,0 +1,72 @@
+    items = SaleItemInputSerializer(many=True, write_only=True)
+    
+    # Read-only nested representation (for viewing the receipt)
+    sale_items = serializers.SerializerMethodField()
+    
+    class Meta:
+        model = Sale
+        fields = [
+            'id', 'reference', 'customer', 'currency', 'exchange_rate',
+            'total_amount', 'amount_paid', 'payment_method', 
+            'items', 'sale_items', 'created_at'
+        ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)